commutative-semigroups.cabal

```diff
@@ -1,10 +1,9 @@
-cabal-version:       2.4
```

Comment:
This field is now required, see https://cabal.readthedocs.io/en/3.6/cabal-package.html#package-properties

Reply:
reverted